### ISSUE TITLE
call addEventListener directly

### ISSUE THREAD

**Issue by tpmarc** [lpmourato/musical-note#19]
This const is useless. Calling inline `addEventListener` isn't harmful. 

https://github.com/lpmourato/musical-note/blob/710ace6e9943954a2a06419cf2e5f8de8f1f076f/server/static/scripts/main.js#L26

**Comment by lpmourato** [comment]
you mean document.getElementById('play').addEventListener?

**Comment by tpmarc** [comment]
Yep. Once you using it just to add and event, it could be better add the event directly. But it's your app's play button, then maybe it could be useful later. This decision is all yours 😬

**Comment by lpmourato** [comment]
Ok, I'll keep like that until verify if it is really necessary.

**Comment by lpmourato** [comment]
for now, we have no sure about how useful are those constants.
